### PR TITLE
[material] Fix binding of object schema in tables

### PR DESCRIPTION
**Pull request by edgarmueller**

### CODE REVIEW

**Comment by edgarmueller** [comment]
@eneufeld I went ahead and merged this in order to create another alpha release. Please review it nevertheless, I'll take are of any review comments.

File: packages/material/src/complex/MaterialTableControl.tsx [hunk 1]
```diff
@@ -131,33 +134,47 @@ const TableHeaderCell = ({ propName }: TableHeaderCellProps) => (
...
-        <DispatchField
-          schema={scopedSchema}
-          uischema={Generate.controlElement(
-            undefined,
-            scopedSchema.type === 'object' ? `#/properties/${propName}` : '#'
-          )}
-          path={path}
-        />
+        {
+          schema.properties ?
```
Comment (eneufeld) [inline packages/material/src/complex/MaterialTableControl.tsx]: could you double check the format?

File: packages/material/test/renderers/MaterialArrayControl.test.tsx [hunk 2]
```diff
@@ -215,4 +215,58 @@ describe('Material array control', () => {
...
+      <Provider store={store}>
+        <MaterialArrayControlRenderer schema={schema} uischema={uischema} />
+      </Provider>
+    );
+
+    const buttons = wrapper.find('button');
+    // 3 buttons
+    // add row
+    // two dialog buttons (no + yes)
+    const nrOfRowsBeforeDelete = wrapper.find('tr').length;
```
Comment (eneufeld) [inline packages/material/test/renderers/MaterialArrayControl.test.tsx]: shouldn't this be nrOfRowsBeforeAdd?

File: packages/material/test/renderers/MaterialArrayControl.test.tsx [hunk 3]
```diff
@@ -215,4 +215,58 @@ describe('Material array control', () => {
...
+
+    const addButton = buttons.at(0);
+    addButton.simulate('click');
+    addButton.simulate('click');
+    wrapper.update();
+    const nrOfRowsAfterDelete = wrapper.find('tr').length;
```
Comment (eneufeld) [inline packages/material/test/renderers/MaterialArrayControl.test.tsx]: And here nrOfRowsAfterAdd?

**Pull request merged**